app/controllers/projects_controller.rb

```diff
@@ -101,16 +101,21 @@ def toggle_status
   def sign_user_out
     user = User.find(params[:user_id])
     @project = Project.find(params[:id])
-    if can?(:edit, @project) || current_user == user
-      @project.remove_user(user)
-      if user == current_user
-        redirect_to projects_path
+    unless user.is_wimi? and @project.wimis.count <= 1
```

Comment:
http://www.railstips.org/blog/archives/2008/12/01/unless-the-abused-ruby-conditional/

Die richtige Verhaltensweise für unless-else ist den else- und den unless-Block zu tauschen und if zu benutzen.